services/crawler/crawler.go

```diff
@@ -48,17 +41,31 @@ func Crawl(searchString string) (data *CrawlData, err error) {
 		return nil, err
 	}
 
-	data = parseCrawledData()
+	searchResult = &models.SearchResult{
+		TopAdWordAdvertisersCount:   getTopAdWordAdvertisersCount(),
+		TopAdWordAdvertisersUrls:    getTopAdWordAdvertisersUrls(),
+		TotalAdWordAdvertisersCount: GetTotalAdWordAdvertisersCount(),
```

Comment:
Is there any specific reason to export only this function? (`GetTotalAdwordAdvertisersCount`) while others are private function.

Reply:
Thanks, good catch! It will be private like others